doc/rados/operations/monitoring-osd-pg.rst

```diff
@@ -26,12 +26,22 @@ An OSD's status is either in the cluster (``in``) or out of the cluster
 running (``down``). If an OSD is ``up``, it may be either ``in`` the cluster
 (you can read and write data) or it is ``out`` of the cluster. If it was
 ``in`` the cluster and recently moved ``out`` of the cluster, Ceph will migrate
+
```

Comment:
hmm, I think the above lines could be phrased better, speaking of being used for placement instead of can read and write